src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/participationhistory/restapi/CourseParticipationIntegrationTest.kt

```diff
@@ -167,13 +163,13 @@ class CourseParticipationIntegrationTest : IntegrationTestBase() {
     getCourseIds().forEach {
       createCourseParticipation(
         courseId = it,
-        prisonNumber = PRISON_NUMBER,
+        prisonNumber = prisonNumber(),
       ).expectStatus().isCreated
         .expectBody<CourseParticipation>()
         .returnResult().responseBody!!
     }
 
-    val randomPrisonNumber = randomStringUpperCaseWithNumbers(7)
```

Comment:
Thinking about it, `randomStringUpperCaseWithNumbers()` was something I'd implemented for use with prison numbers. If we've written something more specified, we may not need this at all now.